# AddObstacle service definition
# Request: obstacle parameters
float64 x               # X position of obstacle
float64 y               # Y position of obstacle
float64 radius          # Obstacle radius
string obstacle_type    # "static", "dynamic"
float64 velocity_x      # X velocity for dynamic obstacles
float64 velocity_y      # Y velocity for dynamic obstacles
---
# Response: success status and message
bool success            # True if obstacle was added successfully
string message          # Status message or error description
int32 obstacle_id       # Unique ID of the added obstacle 